lib/index.js

```diff
@@ -110,7 +119,7 @@ export default {
 
         const messages = HTMLHint.verify(fileText, ruleset || undefined);
 
-        if (editor.getText() !== fileText) {
+        if (removePHP(editor.getText()) !== fileText) {
```

Comment:
This code is checking whether the editor was modified while `HTMLHint` was running, as such it shouldn't be touched. (This check would always fail on any document `removePHP` would modify).

Reply:
I'll store `editor.getText()` twice at the start and use that, then. It can't stay as is because `fileText` has changed.

Reply:
Whoops, was taking that change in isolation, but I like the way you fixed this better anyway 👍.